### ISSUE TITLE
App Image dependency error

### ISSUE THREAD

**Issue by halak0013** [0xzer0x/quran-companion#30]
As-salamu Alaykum,
Thank you for developing this Quran app. Allah cc. bless you.
Wehen i try to run app image [latest release](https://github.com/0xzer0x/quran-companion/releases/tag/v1.1.11) it give dependences error  
`Image: error while loading shared libraries: libb2.so.1: cannot open shared object file: No such file or directory` 
when i install `libb2-1`. this error fixed. but now it give qt6 and libc error

```
.../Applications/Quran_Companion-1.1.11-x86_64_7f57833a2a570367c511a6a70bd8576f.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by /tmp/.mount_Quran_GcJjvX/usr/bin/../lib/libQt6Gui.so.6)
.../Applications/Quran_Companion-1.1.11-x86_64_7f57833a2a570367c511a6a70bd8576f.AppImage: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by /tmp/.mount_Quran_GcJjvX/usr/bin/../lib/libQt6Gui.so.6)
.../Applications/Quran_Companion-1.1.11-x86_64_7f57833a2a570367c511a6a70bd8576f.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by /tmp/.mount_Quran_GcJjvX/usr/bin/../lib/libQt6Network.so.6)
```
my system ([Pardus](https://www.pardus.org.tr/en/) it based debian 12) has libc6 2.36-9

**Comment by 0xzer0x** [comment]
Walaikum Assalam,
Thanks for reporting the issue. It was an error in the Qt libraries used. It should be fixed now (tested on debian 12 and ubuntu  22.04).

**Comment by halak0013** [comment]
`sudo apt install libxcb-cursor0` also there is a small bug. yesterdey i reinstall my system and try to run appimage. it didn't work. i installed `libxcb-cursor0` and now it is working.
